### PR TITLE
Add image resizedHeight model attribute (convert from style height)

### LINKED ISSUE #14357: Add image resizedHeight attribute, convert from style=height:xx
## Provide a description of the task

1. Convert `style=height:xx` to model’s `resizedHeight`.
2. After resize:

   - option 1: remove `resizedHeight`, change `resizedWidth`
   - option 2: change both  `resizedHeight & resizedWidth`

## 📃 Other details

* Browser: …
* OS: …
* CKEditor version: …
* Installed CKEditor plugins: …

### PR DESCRIPTION
**Pull request by mmotyczynska**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Feature (image): Add image `resizedHeight` model attribute converted from image `height` style. Closes #14357.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

File: packages/ckeditor5-image/src/imageresize/imageresizeediting.ts [hunk 1]
```diff
@@ -97,6 +97,7 @@ export default class ImageResizeEditing extends Plugin {
 	 */
 	private _registerConverters( imageType: 'imageBlock' | 'imageInline' ) {
 		const editor = this.editor;
+		const imageUtils = editor.plugins.get( 'ImageUtils' );
```
Comment (arkflpc) [inline packages/ckeditor5-image/src/imageresize/imageresizeediting.ts:100]:
```suggestion
		const imageUtils: ImageUtils = editor.plugins.get( 'ImageUtils' );
```

`ImageUtils` should be imported to make type inference work here. I know it is, but as a matter of style, enforce the need of importing it here.
I know it's imported, but let's enforce

**Review by arkflpc — CHANGES_REQUESTED** [review]

File: packages/ckeditor5-image/tests/imagesizeattributes.js [hunk 2]
```diff
@@ -63,87 +64,143 @@ describe( 'ImageSizeAttributes', () => {
 			describe( 'inline images', () => {
 				it( 'should upcast width attribute correctly', () => {
 					editor.setData(
-						'<p>Lorem <img width="100px" src="/assets/sample.png" "> ipsum</p>'
+						'<p>Lorem <img width="100" src="/assets/sample.png" "> ipsum</p>'
```
[the remark this answers is not in the document]
Comment (mmotyczynska) [reply packages/ckeditor5-image/tests/imagesizeattributes.js:67]: Width & height attributes must be an integer without a unit. These attributes represent width & height of the image, in pixels.

File: packages/ckeditor5-image/src/imageresize/imageresizeediting.ts (hunk 1 above)
Comment (mmotyczynska) [reply packages/ckeditor5-image/src/imageresize/imageresizeediting.ts]: Done in 32104408ac4e8910715b4685b369d66a6dda8c34.

File: packages/ckeditor5-image/src/imageresize/imageresizeediting.ts [hunk 3]
```diff
@@ -118,6 +119,36 @@ export default class ImageResizeEditing extends Plugin {
...
+					return;
+				}
+
+				const viewWriter = conversionApi.writer;
+				const figure = conversionApi.mapper.toViewElement( data.item );
+				const img = imageUtils.findViewImgElement( figure );
+				const target = imageType === 'imageInline' ? img : figure;
+
+				if ( data.attributeNewValue !== null ) {
+					viewWriter.setStyle( 'height', data.attributeNewValue, target );
```
[the remark this answers is not in the document]
Comment (mmotyczynska) [reply packages/ckeditor5-image/src/imageresize/imageresizeediting.ts:146]: Done in 19855582b831458d899a14a594331b292d21c352.

File: packages/ckeditor5-image/src/imageresize/imageresizeediting.ts [hunk 4]
```diff
@@ -128,7 +159,39 @@ export default class ImageResizeEditing extends Plugin {
 				},
 				model: {
 					key: 'resizedWidth',
-					value: ( viewElement: ViewElement ) => viewElement.getStyle( 'width' )
+					value: ( viewElement: ViewElement ) => {
+						const widthStyle = imageUtils.getSizeInPx( viewElement.getStyle( 'width' ) );
+						const heightStyle = imageUtils.getSizeInPx( viewElement.getStyle( 'height' ) );
+
+						if ( widthStyle && heightStyle ) {
```
[the remark this answers is not in the document]
Comment (mmotyczynska) [reply packages/ckeditor5-image/src/imageresize/imageresizeediting.ts:168]: You're absolutely right. Done in 4decf192d5c0d64c14b16a55c75963bc1386a10c.

File: packages/ckeditor5-image/src/imageresize/imageresizehandles.ts [hunk 5]
```diff
@@ -63,6 +64,7 @@ export default class ImageResizeHandles extends Plugin {
 	private _setupResizerCreator(): void {
 		const editor = this.editor;
 		const editingView = editor.editing.view;
+		const imageUtils = editor.plugins.get( 'ImageUtils' );
```
[the remark this answers is not in the document]
Comment (mmotyczynska) [reply packages/ckeditor5-image/src/imageresize/imageresizehandles.ts]: Done in 662472bc6059ad21a7938bc972ceb172666e77eb

File: packages/ckeditor5-image/tests/manual/imagesizeattributesallcases.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+* `width` (image width attribute)
+* `height` (image height attribute)
+* `resizedWidth` (image width style)
+* `resizedHeight` (image height style)
+
+Image in the editor should look like the image next to the editor (created from editor's output data):
+* after initial editor load
+* after resizing image in the editor
+
+**Note**: Every time an image is resized, the code below is updated with refreshed output data and model.
```
[the remark this answers is not in the document]
Comment (mmotyczynska) [reply packages/ckeditor5-image/tests/manual/imagesizeattributesallcases.md]: I changed it so it's more clear (I hope). Done in 9d7fda9736481d527c688a1d3a44030541c4fb8a.

**Review by arkflpc — APPROVED** [review]

**Pull request merged**